src/server/routers/tips.ts

```diff
@@ -116,7 +116,13 @@ export const tipsRouter = router({
 
       throw new Error('Unauthorized')
     }),
+  published: baseProcedure
+    .input(z.object({limit: z.number().optional()}).optional())
+    .query(async ({input}) => {
+      const tips = await getAllTips({onlyPublished: true, limit: input?.limit})
 
+      return tips
+    }),
```

Comment:
leaving `tips.all` alone for the moment and adding this `tips.published` route for the Tips Sanity Section which only grabs a limited number of published tips, rather than all tips and then processing after the fact